src/features/device-heartbeat/index.ts

```diff
@@ -122,7 +122,7 @@ export const getPollInterval = async (uuid: string) => {
 export const POLL_JITTER_FACTOR = 1.5;
 
 // these align to the text enums coming from the SBVR definition of available values...
-export const enum DeviceOnlineStates {
```

Comment:
We can't re-export const enums as part of the `device` object, since they are supposed to be replaced with string values on build time.
See: https://app.circleci.com/pipelines/github/balena-io/open-balena-api/5280/workflows/946eb8f6-a24b-453e-8661-955cbd9a3a77/jobs/4976

I would expect that this wouldn't be an issue, given that we use `preserveConstEnums` in our tsconfig, but it ends up that TS still complains.